AI/MIGraphX/Quantization/Running Quantized ResNet50 via MIGraphX.md

```diff
@@ -0,0 +1,104 @@
+
+
+# Running quantized ResNet50 via MIGraphX
+
+## Summary
+This example walks through the dynamo Post Training Quantization (PTQ) workflow for running a quantized model using torch_migraphx.
+
+## Prerequisites
+You must follow the installation instructions for the torch_migraphx library in [Readme.md](https://github.com/Rmalavally/rocm-examples/blob/develop/AI/MIGraphX/Quantization/Readme.md) before using this example.
```

Comment:
personal forks should not be linked to.

Reply:
Will update this once we have the approval to merge.

Reply:
Added. Please review the Readme file in rocm-examples.